### PR TITLE
Fix compilation warning in s2builderutil_lax_polygon_layer_test.cc

### PR DESCRIPTION
**Pull request by r-barnes**
Fixes the issue described in #114

### CODE REVIEW

File: src/s2/s2builderutil_lax_polygon_layer_test.cc [hunk 1]
```diff
@@ -52,6 +53,8 @@ string ToString(DegenerateBoundaries degenerate_boundaries) {
     case DegenerateBoundaries::DISCARD_HOLES: return "DISCARD_HOLES";
     case DegenerateBoundaries::DISCARD_SHELLS: return "DISCARD_SHELLS";
     case DegenerateBoundaries::KEEP: return "KEEP";
+    default:
+      throw std::runtime_error("Unrecognized DegenerateBoundaries value!");
   }
 }
```
Comment (jmr) [inline src/s2/s2builderutil_lax_polygon_layer_test.cc:59]: `s2geometry` doesn't use exceptions, so `return "UNKNOWN";` or `LOG(FATAL) << ...` *after* the switch would be better, but I'm really wondering why you're getting this warning when the cases are exhaustive.

Comment (r-barnes) [reply src/s2/s2builderutil_lax_polygon_layer_test.cc:58]: I'm also a little mystified as to why the compiler wouldn't detect this, but I do get this warning with an unaltered configuration using G++ 9.3.0. I've updated the commit to use `LOG`, as you suggest.

**Comment by r-barnes** [comment]
Not sure if I should have resolved the above conversation. My reply:
> I'm also a little mystified as to why the compiler wouldn't detect this, but I do get this warning with an unaltered configuration using G++ 9.3.0. I've updated the commit to use LOG, as you suggest.

File: src/s2/s2builderutil_lax_polygon_layer_test.cc [hunk 2]
```diff
@@ -53,6 +54,7 @@ string ToString(DegenerateBoundaries degenerate_boundaries) {
     case DegenerateBoundaries::DISCARD_SHELLS: return "DISCARD_SHELLS";
     case DegenerateBoundaries::KEEP: return "KEEP";
   }
+  S2_LOG(FATAL) << "Unrecognized DegenerateBoundaries type!";
```
Comment (jmr) [inline src/s2/s2builderutil_lax_polygon_layer_test.cc:57]: How about
```
S2_LOG(FATAL) << "Unknown DegenerateBoundaries value: "
              << static_cast<int>(degenerate_boundaries);
```
?

I can also just fix it if you want.

File: src/s2/s2builderutil_lax_polygon_layer_test.cc [hunk 3]
```diff
@@ -21,6 +21,7 @@
 #include <map>
 #include <memory>
 #include <set>
+#include <stdexcept>
```
[the remark this answers is not in the document]
Comment (jmr) [reply src/s2/s2builderutil_lax_polygon_layer_test.cc:24]: This was never fixed, so merging #136 instead.

**Pull request closed without merging**